packages/fxa-settings/src/components/FlowSecondaryEmailVerify/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { gql, useMutation } from '@apollo/client';
+import sentryMetrics from 'fxa-shared/lib/sentry';
+import TextInput from '../TextInput';
+import { RouteComponentProps, useNavigate } from '@reach/router';
+import FlowContainer from '../FlowContainer';
+import { Account } from '../../models';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
+
+export const VERIFY_SECONDARY_EMAIL_MUTATION = gql`
+  mutation verifySecondaryEmail($input: VerifyEmailInput!) {
+    verifySecondaryEmail(input: $input) {
+      clientMutationId
+    }
+  }
+`;
+
+export const FlowSecondaryEmailVerify = ({ location }: RouteComponentProps) => {
+  const [code, setCode] = useState<string>();
+  const [errorText, setErrorText] = useState<string>();
+  const goBack = useCallback(() => window.history.back(), []);
+  const navigate = useNavigate();
+  const email = (location?.state as any)?.email as string | undefined;
+
+  const [verifySecondaryEmail] = useMutation(VERIFY_SECONDARY_EMAIL_MUTATION, {
```

Comment:
I had trouble using the new `useHandledMutation` function. The `setErrorText` wasn't triggering the ui to update. I haven't debugged it yet.

Reply:
Interesting - FWIW I have it working in a similar fashion here https://github.com/mozilla/fxa/pull/6292/files#diff-a313ff76f12b71cf55da0fc953b5dabbR88-R91

Reply:
Fixed it, it was calling itself https://github.com/mozilla/fxa/pull/6292/files#diff-ed45386ca794096a72b565395c90232dR75-R89